resources/profiles/Anker/process/0.15mm Optimal @Anker.json

```diff
@@ -0,0 +1,20 @@
+{
+    "type": "process",
+    "setting_id": "GPGP_Anker_003",
```

Comment:
setting_id is not valid

Reply:
@XunZhangBambu Please assign a right one

Reply:
@XunZhangBambu Please assign a right one

Reply:
changed as requested